tests/system/source/main.d

```diff
@@ -88,6 +114,143 @@ void main ()
         checkBlockHeight(1);
     }
 
+    /// Make Transactions for block height 2
+    {
+        API[NodeCnt] clients;
+        foreach (idx, const ref addr; Addrs)
+            clients[idx] = new RestInterfaceClient!API(addr.withSchema());
+
+        // clients[0] Create frozen transactions and enrolment
+        const Block[] blocks_1 = clients[0].getBlocksFrom(1,1);
+        assert(blocks_1[0].header.height == 1);
+
+        foreach (idx; 0 .. Block.TxsInBlock)
+        {
+            Transaction tx;
+
+            if (keypairs.length > idx)
+            {
+                Transaction freeze_tx = {
+                    type: TxType.Freeze,
+                    inputs: [Input(hashFull(blocks_1[0].txs[idx]), 0)],
+                    outputs: [Output(Amount.MinFreezeAmount,
+                        keypairs[idx].address)]
+                };
+                freeze_tx.inputs[0].signature = kp.secret.sign(hashFull(freeze_tx)[]);
+                clients[0].putTransaction(freeze_tx);
+                freeze_txs ~= freeze_tx;
+            }
+            else
+            {
+                Transaction payment_tx = {TxType.Payment, [], []};
+                payment_tx.inputs ~= Input(hashFull(blocks_1[0].txs[idx]), 0);
+                foreach (idx2; 0 .. Block.TxsInBlock)
```

Comment:
It would have been simpler if there was only one Output that had the sum of the freeze amounts. So `MinFreezeAmount * Block.TxsInBlock`.

However, `Amount` does not have `operator *` or `multiply()` implemented, and using `Amount.add()` is more work. So you can keep it this way for now.

Reply:
That's right. I've looked up the `operator *`, `multiply()` in Amount, but it hasn't been implemented yet.
I think it is because there is still a risk of implementation.

Reply:
we can treat it as an integer and a decimal point.
But So I can keep it this way for now.